h/templates/admin/oauthclients_create.html.jinja2

```diff
@@ -4,5 +4,20 @@
 {% set page_title = 'Create OAuth client' %}
 
 {% block content %}
+<h3>Security Warning</h3>
+
+<p>Be careful and thoughtful when creating OAuth clients, as these credentials
+give significant power to their users. Some guidelines:</p>
+
+<ul>
+  <li>Do not store these credentials in unencrypted form; share them securely only with their intended users.</li>
```

Comment:
@robertknight I'm wondering if I should back off and  only have text text in about `client_credentials` grant type (not mentioning the other kinds), in the interest of not making this a big task. I've been looking at source code but I'm still a bit wiggly about `jwt_bearer`. Thoughts?

Reply:
> @robertknight I'm wondering if I should back off and only have text text in about client_credentials grant type (not mentioning the other kinds), in the interest of not making this a big task.

That sounds reasonable to me. `jwt_bearer` is also a big deal because it allows performing any API request as a particular user without their approval, since anyone with a `jwt_bearer` credential can generate a grant token and use it to obtain an access token.